src/BloomExe/Book/BookStorage.cs

```diff
@@ -246,6 +247,59 @@ public void Save()
 			MetaData.Save();
 		}
 
+		/// <summary>
+		/// Compare the images we find in the top level of the book folder to those referenced
+		/// in the dom, and remove any unreferenced on
+		/// </summary>
+		public void CleanupUnusedImageFiles()
+		{
+			//Collect up all the image files in our book's directory
+			var imageFiles = new List<string>();
+			var imageExtentions = new HashSet<string>(new []{".jpg",".png",".svg"});
+			var ignoredFilenameStarts = new HashSet<string>(new [] {"thumbnail","placeholder"});
+			foreach (var path in Directory.EnumerateFiles(this._folderPath).Where(
+				s => imageExtentions.Contains(Path.GetExtension(s).ToLowerInvariant())))
+			{ 
+				var filename = Path.GetFileName(path);
+				if (ignoredFilenameStarts.Any(s=>filename.StartsWith(s, StringComparison.InvariantCultureIgnoreCase)))
+					continue;
+				imageFiles.Add(Path.GetFileName(GetNormalizedPathForOS(path)));
+			}
+			//Remove each image actually in use from that list
+			foreach (XmlElement img in HtmlDom.SelectChildImgAndBackgroundImageElements(Dom.RawDom.DocumentElement))
+			{
+				var path = GetNormalizedPathForOS(HtmlDom.GetImageElementUrl(img).PathOnly.NotEncoded);
+				imageFiles.Remove(path);   //Remove just returns false if it's not in there, which is fine
+			}
+			//Delete any files still in the list
```

Comment:
Placeholder is in the list of stuff we don't  bother to delete. But if wasn't there, it would just get replaced as needed by some other code.